pom.xml

```diff
@@ -284,12 +298,14 @@
                     <minimumJavaVersion>1.8</minimumJavaVersion>
                     <pluginFirstClassLoader>true</pluginFirstClassLoader>
                     <maskClasses>
```

Comment:
Is there a reason this plugin is using _both_ of these systems? That seems surprising.

Reply:
Erm I’ll try remove plugin first class loader, it doesn’t work on its own without mask classes iirc

Reply:
Wasn't able to remove `pluginFirstClassLoader` was getting weird conflicts with `okhttp`.

But removing `maskClasses` worked...